### ISSUE TITLE
Weather

### ISSUE THREAD

**Issue by Funnytom118** [Funnytom118/Valkine#17]
Allow an option to permanently make it sunny (not rainy/thunder/snow). Should be in options section.

**Comment by Osto468** [comment]
Is that even an option in the original essentials? anywho, might find a way - if I am bothered ;)

**Comment by Valzar** [comment]
Isn't that already a thing from Essentials? /sun -
From my experiences it makes it "permanently" sunny until you change the weather.

**Comment by Funnytom118** [comment]
No, there isn't one, but it's a feature I'd like to see added. You could just check for sun/rain etc.

Also, on 1.11 (or whatever the latest version of MC is), there is a /gamerule for keeping the weather the same, so I thought it would be best to have it in this for older versions.
